.github/workflows/build.yml

```
@@ -40,6 +40,17 @@ jobs:
       with:
         python-version: ${{ env.PYTHON_VERSION }}
 
+    - name: Manual Installation Steps (macOS)
+      if: ${{ runner.os == 'macOS' }}
+      run: |
+        sudo rm -rf /Libary/Developer/CommandLineTools
+        #echo A | softwareupdate --install-rosetta
+        arch -x86_64 xcode-select --install
```

From the tome of ancient wisdom (node-gyp documentation https://github.com/nodejs/node-gyp/blob/0f1f667b737d21905e283df100a2cb639993562a/macOS_Catalina.md#i-did-all-that-and-the-acid-test-still-does-not-pass--): 

```suggestion
        sudo rm -rf $(xcode-select -print-path)
        sudo rm -rf /Libary/Developer/CommandLineTools
        sudo xcode-select --reset
        arch -x86_64 xcode-select --install
```

Hope this works maybe 🤞 . And sorry for all the code suggestions on this PR!

---

Sorry, just took it from the link you posted on Discord, didn't see this beforehand